server/spec/post_bind_bonus_pool_spec.rb

```diff
@@ -100,6 +123,41 @@
   @hostlimited_bonus_stacked_pool = sub_pools[0]
   @hostlimited_bonus_stacked_pool['quantity'].should == 90
 
+  @unlimited_product_master_pool = create_pool_and_subscription(owner['key'], unlimited_prod['id'], -1)
+  # Limit master pool quantity to -1 (Unlimited Quantity)
+  @cp.get_pool(@unlimited_product_master_pool['id'])['quantity'].should == -1
+  pools = @cp.list_owner_pools(@owner_key)
+  pools.length.should == 12
+  sub_pools = pools.select do |pool|
+    pool['subscriptionId'] == @unlimited_product_master_pool['subscriptionId'] &&
+      pool['type'] != 'NORMAL'
+  end
+  sub_pools.length.should == 1
+  @unlimited_product_bonus_pool = sub_pools[0]
+  @cp.get_pool(@unlimited_product_bonus_pool['id'])['quantity'].should == 4
```

Comment:
same here: if master pool is unlimited (-1), the bonus pool should be unlimited (-1)

Reply:
Fixed.